src/views/PillarAnalysis/index.tsx

```diff
@@ -52,6 +77,13 @@ function PillarAnalysis(props: PageProps) {
     } = props;
 
     const [value, setValue] = useState<string | undefined>();
+    const [activeTab, setActiveTab] = useState<TabNames>('entries');
+    const [filtersValue, setFiltersValue] = useState<FaramValues>({});
+    const [entries, setEntries] = useState<Partial<EntryFields>[]>([]);
```

Comment:
why are we using Partial here?

Reply:
I've only requested items required using queries. Is this a bad approach?

Reply:
Maybe use pick? 
CC @tnagorra